### DISCUSSION TITLE
Options to speed up libigl ?

### DISCUSSION THREAD

**Discussion by boxerab** [libigl/libigl#2066] [General]
Hello,
I have a project based on libigl, and I am finding that the windows version is quite a bit slower than on Linux.
Any suggestions on how to address this ?

Also, is it possible to offload some of the computations to a GPU ? 

Thanks!

**Comment by alecjacobson** [comment]
I think there's some way to do Eigen on the GPU, but maybe it'd be non-trivial to port libigl functions to do that. In any case, it'd probably have to be case-by-case.

Often windows is slower because setting the Release + Compiler optimization flags is more confusing. I'd start by ensuring that they're really on.

I also have heard that CGAL can be slower on windows, so if it's a question about the igl::copyleft::cgal functions that could be why.

**Comment by boxerab** [comment]
my current solution for windoze is to not do it at all and to run on `wsl`
